src/tests/end2end/TanhTests.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021 The WebNN-native Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "src/tests/WebnnTest.h"
+
+class TanhTests : public WebnnTest {};
+
+TEST_F(TanhTests, TanhWith1DTensor) {
```

Comment:
Could you please refactor these test align with [webnn-polyfill](https://github.com/webmachinelearning/webnn-polyfill/blob/master/test/ops/tanh.js) so that i can change the api easily.

Reply:
Complete refactoring, PTAL.